changelog/unreleased/issue-4658

```diff
@@ -0,0 +1,8 @@
+Change: Updated Azure Blob Storage Authentication documentation to use `az login`
```

Comment:
We usually don't write changelog entries for documentation updates, so please remove the file again, sorry.

Reply:
Hey @MichaelEischer, 

I have removed the change log file and also squashed all my commits into a single one.